src/components/DataCollector/utils.js

```diff
@@ -50,8 +50,23 @@ const isSupportedAnchorElement = element => {
   return false;
 };
 
+const isDownloadLink = (downloadLinkQualifier, linkUrl, clickedObj) => {
+  const re = new RegExp(downloadLinkQualifier);
+  return clickedObj.download ? true : re.test(linkUrl.toLowerCase());
+};
+
+const isExitLink = (window, linkUrl) => {
+  const currentHostname = window.location.hostname;
+  if (linkUrl.toLowerCase().indexOf(currentHostname) >= 0) {
```

Comment:
Wouldn't this match?
hostname = example.com
linkUrl = http://example.com.foo.com/bar

Reply:
It would. If the domain name is repeated or included in an outbound exit link it would trigger as an internal link. Although unlikely we can enhance this later and parse the link-url for the domain.